tests/test_format_strings.py

```diff
@@ -16,14 +16,6 @@ def test_new_format_string(v):
     check(v.used_names, ['a', 'b', 'c', 'd', 'locals'])
 
 
-@pytest.mark.skipif(
-    sys.version_info < (3, 6),
-    reason="needs f-string support (added in Python 3.6)")
-def test_f_string(v):
-    v.scan("f'{a}, {b:0d} {c:<30} {d:.2%}'")
```

Comment:
Thanks for the PR! I don't think we need more tests for f-strings. Instead, you can extend the old f-string to include your new examples: `f'{a}, {b:0d} {c:<30} {d:.2%} {e()} {f:{width}.{precision}} {{x:y for (x, y) in ((1, 2), (3, 4))}}'`